src/contour/CMakeLists.txt

```diff
@@ -63,14 +63,25 @@ function(DeclareQmlImport name import version)
     endif()
 endfunction()
 
+function(DeclareQML name name_qt5 name_qt6)
+    if(CONTOUR_QT_VERSION EQUAL "5")
+        set(${name} "${name_qt5}" PARENT_SCOPE)
+    else()
+        set(${name} "${name_qt6}" PARENT_SCOPE)
+    endif()
+endfunction()
+
 DeclareQmlImport(qml_import_ContourTerminal "Contour.Terminal" "1.0")
 DeclareQmlImport(qml_import_QtQuick "QtQuick" "2.6")
-DeclareQmlImport(qml_import_QtQuickControls "QtQuick.Controls" "2.6")
+DeclareQmlImport(qml_import_QtQuickControls "QtQuick.Controls" "2.2")
 DeclareQmlImport(qml_import_QtQuickLayouts "QtQuick.Layouts" "1.11")
 DeclareQmlImport(qml_import_QtQuickWindow "QtQuick.Window" "2.6")
 DeclareQmlImport(qml_import_QtQuicklabs "Qt.labs.platform" "1.1")
 DeclareQmlImport(qml_import_QtMultimedia "QtMultimedia" "5.12")
 DeclareQmlImport(qml_import_QtGraphicalEffects "QtGraphicalEffects" "1.12" "Qt5Compat.GraphicalEffects")
+DeclareQML(qml_audio "Audio" "AudioOutput")
+DeclareQML(qml_media_player "" "audioOutput: bellAudioOutput")
```

Comment:
interesting take. 🤔

Reply:
do you think that we should do this in some other way? my intent was to keep it as simple as possible so we dont polute qml files itslelf

Reply:
i think it's still better than going for our own solution. so staying in the domain of CMake I do not know how to do it better. (I mean, we could introduce named arguments to our own CMake functions/macros, but that would probably be too much)